test/onnx/test_onnx_opset.py

```diff
@@ -369,6 +370,31 @@ def forward(self, x):
         x = torch.randn(20, 16, 50)
         check_onnx_opsets_operator(MyDynamicModel(), x, ops, opset_versions=[9, 10])
 
+    def test_grid_sample(self):
+        n, c, h_in, w_in, h_out, w_out = 1, 1, 3, 2, 2, 4
+        ops = [{"op_name": "GridSample"}]
+        ops = {16: ops}
+
+        class MyModule(Module):
+            def forward(self, x, grid, mode, padding_mode, align_corers):
+                return torch.nn.functional.grid_sample(x, grid, mode, padding_mode, align_corners)
+
+        for mode, padding_mode, align_corners in itertools.product(
```

Comment:
Possible to parameterize this test? E.g. 
https://github.com/pytorch/pytorch/blob/b34b192d6b97325c9f78e5995c48c8498ede34bd/test/distributed/fsdp/test_fsdp_core.py#L141-L155

Reply:
locally `parametrize` didn't work so skipped it this time

Reply:
also, I'm not sure if we want a bunch of different test cases for one GridSample op.

Reply:
That's cool. I wouldn't worry about having different tests for an op, as long as it's clear what they are testing for. For now I would also recommend subtests: https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests

Reply:
I'm not that inclined to use that as that's not supported by pytest IIRC: https://docs.pytest.org/en/6.2.x/unittest.html#unittest-testcase-support

Reply:
Do you know if pytest will then skip the subtests or treat them as one test instead? It seems they provide support via a plugin: https://github.com/pytest-dev/pytest-subtests. Feel free to skip any changes in this PR!

Reply:
IIRC it was more similar to the latter when I used pytest for other tests with subtest.
I haven't checked the code nor sought for a plugin though